generators/upgrade/index.js

```diff
@@ -82,22 +84,24 @@ module.exports = UpgradeGenerator.extend({
             const generatorDir = this.clientPackageManager === 'yarn' ? shelljs.exec('yarn bin', { silent: this.silent }).stdout : shelljs.exec('npm bin', { silent: this.silent }).stdout;
             generatorCommand = `${generatorDir.replace('\n', '')}/jhipster`;
         }
-        shelljs.exec(`${generatorCommand} --with-entities --force --skip-install`, { silent: this.silent }, (code, msg, err) => {
-            if (code === 0) this.log(chalk.green(`Successfully regenerated application with JHipster ${version}`));
+        const regenerateCmd = `${generatorCommand} --with-entities --force --skip-install`;
+        this.log.info(regenerateCmd);
```

Comment:
No because this.log.info() provides 2 things I like here: indentation and grey info prefix like you can see in [screenshot](https://github.com/jhipster/generator-jhipster/issues/6237#issuecomment-323235768).